Merge conlusions branch in (#8)
ssteinbach/ordinate_precision_research: README.md

See: [results.md](results.md)

## Conclusions

* We have a number of tests that explore the accuracy of a variety of number
  representations present in audio/video engineering, including integer
  rational, and various bit depths of floating point number.
* If your system does not require time warps, than integers, or "factorized

>   integers"/integer rationals (if you're mixing rates) is probably sufficient.

Reviewer: ```suggestion
  integers"/integer rationals (if you're not mixing rates) is probably sufficient.
```

> * Given a system that aims to allow warping and audio rates, however, a

Reviewer: ```suggestion
* Given a system that aims to allow warping,  and audio rates or NTSC rates however, a
```

  floating point number may be a better choice.
* Floating point numbers do not suffer the same accuracy penalties under

>   multiplication that integer rationals do

Reviewer: ```suggestion
  multiplication that integer rationals do.
```

* In the limit, integer rational implementations reinvent floating point
  numbers without the benefit of hardware acceleration and deep software
  support.
* For a system that uses a floating-point based number, it is possible to
  compute the accuracy limits imposed by the system and the number bit width.
* For systems that are using integers, conversions to floating point numbers

>   are still subject to the limits of floating point accuracy.  IE if you need

Reviewer: ```suggestion
  are still subject to the limits of floating point accuracy.  If you need
```

>   to synchronize a point that is past the floating point accuracy limit, it

Reviewer: ```suggestion
  to synchronize a time value that is past the floating point accuracy limit, it
```

  doesn't matter if you compute the input with integer accuracy if you still
  need to use a floating point number without sufficient accuracy to convert

>   into the other space (IE fetching an audio sample during a long running video

Reviewer: ```suggestion
  into the other space such as occurs when fetching an audio sample during a long running video
```

>   timeline)

Reviewer: ```suggestion
  timeline.
```
